packages/venia-ui/lib/components/MiniCart2/minicart.css

```diff
@@ -0,0 +1,80 @@
+.root {
+    /* The root aside element takes up the whole screen. */
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+
+    /* It is hidden by default. */
+    opacity: 0;
+    visibility: hidden;
+
+    /* It animates to being closed. */
+    transition-duration: 192ms;
+    transition-timing-function: var(--venia-anim-out);
+    transition-property: opacity, visibility;
+
+    /* It sits over all background content. */
+    z-index: 3;
+}
+
+.root_open {
+    composes: root;
+
+    opacity: 1;
+    visibility: visible;
+
+    /* It animates to being open. */
+    transition-duration: 224ms;
+    transition-timing-function: var(--venia-anim-in);
+}
+
+.body {
+    margin-top: 1rem;
```

Comment:
This is adding some weird whitespace above the body. I do think using top spacing is correct. Not sure what to do here:

[![Image from Gyazo](https://i.gyazo.com/4d094a0b93b75396226174965a18303a.gif)](https://gyazo.com/4d094a0b93b75396226174965a18303a)

Reply:
I switched to a grid `gap` in https://github.com/magento/pwa-studio/pull/2494/commits/9e3683be796d4e7c3dff617ec51a57b31b6f5f24. There's still that whitespace but at least it is consistent on the top and bottom now. You're right, I'm not sure what to do about this.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2494/commits/2ae158939c3d230bca21b7cfe67c7a3debb623f1. It uses `padding` now instead, which allows the contents to scroll into the whitespace.